### PR TITLE
Add Duration.JulianYear

### PR DESCRIPTION
**Pull request by ebfortin**
Julian Year are used in astronomy and cosmology as a constant year length.

### CODE REVIEW

File: UnitsNet.NumberExtensions/UnitsNet.NumberExtensions.csproj [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
     <RootNamespace>UnitsNet</RootNamespace>
-    <TargetFrameworks>netstandard2.0;net40</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>
```
[the remark this answers is not in the document]
Comment (ebfortin) [reply UnitsNet.NumberExtensions/UnitsNet.NumberExtensions.csproj]: If we support netstandard2 then 4.7 is automatically supported as the minimum version of the .NET Framework that support netstandard2 if 4.6.1. Ref. : https://docs.microsoft.com/en-us/dotnet/standard/net-standard

I still can't build with NET4 even with NET Framework 4.8 installed. Maybe it has to do with my use of Visual Studio 2022?

File: UnitsNet.NanoFramework/GeneratedCode/AmountOfSubstance/UnitsNet.NanoFramework.AmountOfSubstance.nuspec [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
   <metadata>
     <id>UnitsNet.nanoFramework.AmountOfSubstance</id>
-    <version>4.102.0</version>
+    <version>4.101.0</version>
```
[the remark this answers is not in the document]
Comment (ebfortin) [reply UnitsNet.NanoFramework/GeneratedCode/AmountOfSubstance/UnitsNet.NanoFramework.AmountOfSubstance.nuspec:5]: Yeah had some problem with that. Clicked where I shouldn't have. Thought it was resolved. Seems not.

File: UnitsNet.Serialization.JsonNet/UnitsNet.Serialization.JsonNet.csproj [hunk 3]
```diff
@@ -23,7 +23,7 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion> <!-- Should reflect major part of Version -->
     <LangVersion>latest</LangVersion>
     <RootNamespace>UnitsNet.Serialization.JsonNet</RootNamespace>
-    <TargetFrameworks>netstandard2.0;net40</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net40;net47</TargetFrameworks>
```
Comment (angularsen) [inline UnitsNet.Serialization.JsonNet/UnitsNet.Serialization.JsonNet.csproj:26]: It seems you maybe missed this one. Changing target framework is out of scope for this PR.

Please review the final diff of the PR in the `Files Changed` tab and revert any changes that are not related to adding this unit. Whitespace changes are fine.

https://github.com/angularsen/UnitsNet/pull/978/files

**Review by angularsen — CHANGES_REQUESTED** [review]
Another thing to revert, then this should be ready to go.

**Comment by angularsen** [comment]
[Release UnitsNet/4.103.0 · angularsen/UnitsNet](https://github.com/angularsen/UnitsNet/releases/tag/UnitsNet%2F4.103.0)

**Comment by angularsen** [comment]
Sorry for the delay. Nuget 4.103.0 is on the way.

**Pull request merged**